### DISCUSSION TITLE
Bookmarks Backup and Restore support to local fileystem.

### DISCUSSION THREAD

**Discussion by Universalizer** [Alex313031/Thorium-Android#1] [Show and tell]
Thank You @Alex313031 for Thorium Special Android Releases Repository here on,

For me as a end user, most basic feature of / for bookmarks `import and export to Local Filesystem` as 'bookmarks.html'

Approx. 4500 bookmarks from Bromite Browser, even though, now i can import to Cromite, but still if i can also import to Thorium Browser.

'bookmarks.html' can be opened & loaded to any browser, but manual bookmarking it, each & every link, these way takes too much time and energy.

Reasons :-
Not relying on 1) Sync to Google Account (Profiling) Or 2) Any 3rd party Cloud-based sign-in or login service (Profiling),

3) Privacy ; Complete control over end user's data.

Can you pull request or merge request, bookmarks patches from any one of the following Chromium forks :-
1) Bromite or Cromite Browser, GNU General Public License v3.0, Or
2) Kiwi Browser,  BSD-3-Clause license, Or
3) Brave Browser, Mozilla Public License 2.0.

Some references
https://bugzilla.mozilla.org/show_bug.cgi?id=1806482

Thank You In Advance, Well-Wishes for Dedications.

**Comment by gz83** [comment]
Thanks for the suggestion, this feature looks great, we'll discuss it

cc：Alex313031

@Alex313031

**Reply by Universalizer** [reply]
Thanks for response, @gz83 & @Alex313031, please consider it seriously.

That manual bookmarking is overwork & tiredness.
